lib/pynput/keyboard/_xorg.py

```diff
@@ -513,7 +574,8 @@ class Listener(ListenerMixin, _base.Listener):
         KEYPAD_KEYS['KP_8']: KeyCode.from_char('8'),
         KEYPAD_KEYS['KP_9']: KeyCode.from_char('9'),
         KEYPAD_KEYS['KP_Add']: KeyCode.from_char('+'),
-        KEYPAD_KEYS['KP_Decimal']: KeyCode.from_char(','),
+        KEYPAD_KEYS['KP_Begin']: KeyCode.from_char('Begin'),
+        KEYPAD_KEYS['KP_Decimal']: KeyCode.from_char('.'),
```

Comment:
This change highlights an issue with this definition: this depends on the decimal marker used for the input language; on my system, the current definition is correct and this change breaks the library, but I suppose that the current definitions breaks the library for most other systems.